test/e2e/traces_noisy_span_filter_test.go

```diff
@@ -0,0 +1,130 @@
+//go:build e2e
+
+package e2e
+
+import (
+	"fmt"
+	"net/http"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/kyma-project/telemetry-manager/internal/otelcollector/ports"
+	kitk8s "github.com/kyma-project/telemetry-manager/test/testkit/k8s"
+	kitkyma "github.com/kyma-project/telemetry-manager/test/testkit/kyma"
+	kittrace "github.com/kyma-project/telemetry-manager/test/testkit/kyma/telemetry/trace"
+	"github.com/kyma-project/telemetry-manager/test/testkit/mocks/backend"
+	"github.com/kyma-project/telemetry-manager/test/testkit/mocks/urlprovider"
+	kittraces "github.com/kyma-project/telemetry-manager/test/testkit/otlp/traces"
+	"github.com/kyma-project/telemetry-manager/test/testkit/verifiers"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/kyma-project/telemetry-manager/test/testkit/periodic"
+)
+
+var _ = Describe("Filter Noisy Trace Spans", Label("tracing"), func() {
+
+	const (
+		mockBackendName = "traces-filter-receiver"
+		mockNs          = "traces-filter-test"
+	)
+
+	var (
+		pipelineName string
+		urls         = urlprovider.New()
+	)
+
+	makeResources := func() []client.Object {
+		var objs []client.Object
+
+		objs = append(objs, kitk8s.NewNamespace(mockNs).K8sObject())
+
+		mockBackend := backend.New(mockBackendName, mockNs, backend.SignalTypeTraces, backend.WithPersistentHostSecret(true))
+		objs = append(objs, mockBackend.K8sObjects()...)
+		urls.SetMockBackendExport(mockBackend.Name(), mockBackend.TelemetryExportURL(proxyClient))
+
+		pipeline := kittrace.NewPipeline(fmt.Sprintf("%s-pipeline", mockBackend.Name())).
+			WithOutputEndpointFromSecret(mockBackend.HostSecretRef()).
+			Persistent(isOperational())
+		pipelineName = pipeline.Name()
+		objs = append(objs, pipeline.K8sObject())
+
+		urls.SetOTLPPush(proxyClient.ProxyURLForService(
+			kitkyma.SystemNamespaceName, "telemetry-otlp-traces", "v1/traces/", ports.OTLPHTTP),
+		)
+
+		traceGatewayExternalService := kitk8s.NewService("telemetry-otlp-traces-external", kitkyma.SystemNamespaceName).
```

Comment:
We don't need an external service anymore, we can call ClusterIP services using the proxyClient

Reply:
removed